### PR TITLE
Apply policy to host forwarded traffic

### PR DESCRIPTION
**Pull request by bmckercher123**
New How to.

- Can merge at any time.
- Approved by Casey and Spike.
- Requires sanity check.

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 2b0a35d6a8b9366a74c3cb893e77e184378008cd

https://deploy-preview-2766--calico.netlify.com

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 24bc81552717cdd44cc9c28cfefcb080f82622b6

https://deploy-preview-2766--calico.netlify.com

File: master/security/host-forwarded-traffic.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+#### Workload endpoints and host endpoints
+
+The following figure shows a host with two network interfaces: eth0 and eth1. We call these **host endpoints (HEPs)**. The host also runs two guest workloads (VMs or containers). We call the virtual interfaces to the guests, **workload endpoints (WEPs)**.  Each has a corresponding configuration object on the {{site.prodname}} API called HostEndpoint and WorkloadEndpoint, respectively. 
+
+The `HostEndpoint API` object is optional, and Calico does not enforce any policy on the HEP if the API object is missing. The `WorkloadEndpoint API` object is required, and is automatically managed by the cluster orchestrator plugin (for example, Kubernetes or OpenStack).
+
+Several connections are shown in the figure, numbered 1 through 4. For example, connection 1 ingresses over HEP eth0, is forwarded, and then ingresses Workload A’s WEP. Calico policies select which WEPs or HEPs they apply to. So, for example an ingress policy that selects Workload A’s WEP will apply to connections as shown in number 1.
+
+![Host-forward-traffic]({{site.baseurl}}/images/host-forward-traffic.png)
```
Comment (lmm) [inline master/security/host-forwarded-traffic.md]: image

Looks like a typo here - there should be an `eth0` and an `eth1`.

File: master/security/host-forwarded-traffic.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+#### applyOnForward
+
+By default, {{site.prodname}} global network policies set **applyOnForward to false**. When set to false on policies that select HEPs, the policies are applied only to traffic that originates or terminates on the host, for example: connection 4 (Node process). Connections 1-3 are unaffected by policies that select the HEP, but have applyOnForward set to false.
+
+In contrast, if applyOnForward is set to true for a policy that selects a HEP, that policy can apply to all connections 1-4. For example:
+
+- Ingress policy on HEP eth0 affects connections 1 and 2
+- Egress policy on HEP eth1 affects connections 2, 3, and 4
+
+There are also different default action semantics for forwarded traffic (connections 1-4) versus local traffic (connection 4) when it traverses a HEP.  If no applyOnForward policy selects the HEP and direction (ingress versus egress), then forwarded traffic is allowed.  If no policy (regardless of applyOnForward) selects the HEP and direction, then local traffic is denied.
```
Comment (lmm) [inline master/security/host-forwarded-traffic.md]: > forwarded traffic (connections 1-4) versus local traffic (connection 4) 

Isn't the forwarded traffic just connections 1-3?

File: master/security/host-forwarded-traffic.md (hunk 1 above)
Comment (bmckercher123) [reply master/security/host-forwarded-traffic.md]: Great catch! Fixed.

File: master/security/host-forwarded-traffic.md (hunk 2 above)
Comment (bmckercher123) [reply master/security/host-forwarded-traffic.md]: Yes, that sentence is not clear. Better?

**Pull request merged**